contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseScanSpec.java

```diff
@@ -76,6 +78,10 @@ public String getTableName() {
     return stopRow == null ? HConstants.EMPTY_START_ROW : stopRow;
   }
 
+  public long getRowCount() { return rowCount; }
+
+  public void setRowCount(long numRowCount) { rowCount = numRowCount; }
```

Comment:
Should the ScanSpec be mutable since it serves a specification? I looked at other ScanSpecs but none seem to set ScanSpec members. Maybe all the members should be final and initialized via the constructor?